### ISSUE TITLE
question about loss function

### ISSUE THREAD

**Issue by Yufengevan** [Tencent/MedicalNet#28]
Hi, I'm confused about the loss function in the train phase.The loss function you have used is cross entropy Loss and the tensors inputted are different shape.Maybe something wrong? Meanwhile i found others have encountered the same problem that net perform well in the train phase,it's loss value is small,but dice value in the test phase maybe 0.

**Comment by Abbas009** [comment]
Hello!
Did you found the solution,I also trapped in a similar problem. The dice loss is always zero when testing it on own dataset.
